packages/jest-runtime/src/index.js

```diff
@@ -12,11 +12,19 @@
 
 import type {Config, Path} from 'types/Config';
 import type {Environment} from 'types/Environment';
+import type {HasteResolverContext} from './types';
 
+export type BuildHasteMapOptions = {
```

Comment:
export types should be after requires. The reason why this is, is because they might use some of the required stuff as types, so the code convention we decided on is:

* import type
* require
* custom types/export type